### DISCUSSION TITLE
Military Frequencies

### DISCUSSION THREAD

**Discussion by tet024** [rtl-airband/RTLSDR-Airband#278] [Q&A] [answered]
Hello, 

Is it possible to tune into Military Frequency with RTLSDR-Airband or else its just for the airband range ? I'm asking to be able to listen the Red Arrows during an Airshow. Frequencies  in questions  243.450, 242.200

Thanks, 
Et

**Comment by szpajder** [comment, marked as answer]
You can tune to any frequency from 0 to 2^31 Hz (provided that it is
supported by your receiver).

**Reply by tet024** [reply]
Thanks It Worked :-)

**Comment by feelingwalnut** [comment]
in idiot terms.
dc to roughly 2.1ghz?

**Comment by szpajder** [comment]
Yes.

**Reply by feelingwalnut** [reply]
Thank you for validating my math.
I had to look up what a ^ was.
